conda_smithy/utils.py

```diff
@@ -60,7 +61,9 @@ def render_meta_yaml(text):
         )
     )
     mockos = MockOS()
-    content = env.from_string(text).render(os=mockos, environ=mockos.environ)
+    py_ver = '.'.join([str(v) for v in sys.version_info[:2]])
```

Comment:
Can you make this `3.7` for all python versions? We shouldn't make it depend on python version that conda-smithy is running

Reply:
Yes, done.

Reply:
I think you forgot to push :smiley: